builders/build/historical-updates.md

```diff
@@ -854,3 +854,31 @@ This migration was executed at the following runtimes and blocks:
 | Moonbase Alpha |      RT1502      |    2112058    |
 
 For more information, you can review the [relative Nimbus PR](https://github.com/moonbeam-foundation/nimbus/pull/45){target=\_blank} and [Moonbeam PR on GitHub](https://github.com/moonbeam-foundation/moonbeam/pull/1400){target=\_blank}.
+
+### Foreign Assets Migration {: #foreign-assets-migration }
+
+A migration was applied to transition existing foreign assets to a new design that manages XCM derivative assets on Moonbeam through EVM smart contracts instead of the previous implementation using the `asset-manager` and `pallet-asset` pallets.
+The migration process involved several extrinsics in the `pallet-moonbeam-lazy-migration`:
+
+- `approve_assets_to_migrate` - Sets the list of asset IDs approved for migration
+- `start_foreign_asset_migration` - Initiates migration for a specific foreign asset by freezing the original asset and creating a new EVM smart contract
+- `migrate_foreign_asset_balances` - Migrates asset balances in batches from old assets pallet to the new system
+- `migrate_foreign_asset_approvals` - Migrates asset approvals in batches while unreserving deposits from the old approval system
+- `finish_foreign_asset_migration` - Completes migration after all balances and approvals are migrated and performs final cleanup
+
+The new foreign assets implementation offers several advantages over the previous system:
+
+- Enables protocols to use Ethereum ecosystem tools (like Foundry)
+- Reduces gas costs for certain operations (e.g., approve)
+- Decreases runtime size by removing two pallets
+- Simplifies runtime code for better maintenance
```

Comment:
Not sure about this. I feel like we should leave these short and sweet and just focus on what data the migration impacted. But this is helpful 🤷‍♀️